library/modules/Items.cpp

```diff
@@ -825,7 +825,7 @@ std::string Items::getDescription(df::item *item, int type, bool decorate)
         addQuality(tmp, item->getQuality());
 
         if (item->isImproved()) {
-            tmp = "<" + tmp + ">";
+            tmp = static_cast<char>(174) + tmp + static_cast<char>(175);
```

Comment:
these as well should be given names so they don't appear to be "magic numbers". e.g. up under `quality_table`, you could add:
```cpp
static const char IMPROVEMENT_MARKER_LEFT = static_cast<char>(174);
static const char IMPROVEMENT_MARKER_RIGHT = static_cast<char>(175);
```

Reply:
roger. I'll go and try making everything work in a more readable way.
thanks for the tips by the way, both of you!

Reply:
As above, these could probably just be `'\xAE'` and `'\xAF'` without having to deal with all of the casts.

Reply:
needless to say, i only saw your comments after posting my last commit 🙄 i'll do another one later as soon as i can